tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -474,8 +474,9 @@ def do_rolling_bounce(self, processor, counter, current_generation):
                     monitors[first_other_processor] = first_other_monitor
                     monitors[second_other_processor] = second_other_monitor
 
-                    version_probing_message = "Sent a version 10 subscription and got version 10 assignment back (successful version probing). Downgrade subscription metadata to commonly supported version 10 and trigger new rebalance.",
-                    end_of_upgrade_message = "Sent a version 10 subscription and group.s latest commonly supported version is 11 (successful version probing and end of rolling upgrade). Upgrading subscription metadata version to 11 for next rebalance."
+                    highest_version = 10
```

Comment:
Could you add comments about why/when we need to bump this value? That can helps us from debugging this issue (again) in the future.

Reply:
Oh I actually added a comment in `StreamsAssignmentProtocolVersions` that describes everything you need to do when bumping the protocol number, and when you should/shouldn't bump it. It should cover everything but now the 1st instruction should be updated to say "bump the `highest_version` variable" instead of saying to update these messages directly @cadonna

Reply:
Good idea! I updated the instructions.